packages/internal.foundry.core/src/_components.ts

```diff
@@ -100,12 +100,7 @@ export interface LteQueryV2 {
 /**
  * Log Safety: UNSAFE
  */
-export interface DeleteLinkRule {
-  linkTypeApiNameAtoB: LinkTypeApiName;
-  linkTypeApiNameBtoA: LinkTypeApiName;
-  aSideObjectTypeApiName: ObjectTypeApiName;
-  bSideObjectTypeApiName: ObjectTypeApiName;
-}
+export type ObjectPrimaryKey = Record<PropertyApiName, PropertyValue>;
```

Comment:
I did not do anything to modify this, or some of the other items below, assuming the spec we generate from was just modified?